common/types/shared.go

```diff
@@ -5572,6 +5688,53 @@ type SignalWithStartWorkflowExecutionRequest struct {
 	JitterStartSeconds                  *int32                 `json:"jitterStartSeconds,omitempty"`
 }
 
+func (v *SignalWithStartWorkflowExecutionRequest) SerializeForLogging() string {
+	// making a deep copy of the struct for manually assigning attributes
+	cloneRequest := SignalWithStartWorkflowExecutionRequest{
```

Comment:
This is not a good pattern. These logic is hard to maintain.
Consider using json encoding to serialize the structs.

Reply:
done